### PR TITLE
CLI: implement delete cmd 

### PR DESCRIPTION
**Pull request by Nexucis**
This PR is proposing a way to delete resources with the CLI. I'm printing here the help from the command, hoping it will help to understand how it is supposed to work and to describe this PR :).

This PR should be merged after #326 because it contains the code to manage file and stdin required for this command.

```bash
./bin/percli delete --help

JSON and YAML formats are accepted.

If both a filename and command line arguments are passed, the command line arguments are used and the filename is
ignored.

Note that the delete command does NOT do resource version checks, so if someone submits an update to a resource right
when you submit a delete, their update will be lost along with the rest of the resource.

Usage:
  percli delete (-f [FILENAME] | TYPE ([NAME1 NAME2] | --all)) [flags]

Examples:

# Delete any kind of resources from a file
percli delete -f data.json

# Delete any kind of resources from stdin
percli data.json | percli delete -f -

# Delete a specific dashboard
percli delete dashboards node_exporter cadvisor

# Delete all dashboards
percli delete dashboards --all


Flags:
  -a, --all              Delete all resources in the project of the specified resource types.
  -f, --file string      Path to the file that contains the resources to delete
  -h, --help             help for delete
  -p, --project string   If present, the project scope for this CLI request

Global Flags:
      --log.level string      Set the log verbosity level. Possible values: panic, fatal, error, warning, info, debug, trace (default "info")
      --percliconfig string   Path to the percliconfig file to use for CLI requests. (default "/Users/ahusson/.perses/config.json")
```

### CODE REVIEW

File: internal/cli/cmd/remove/remove_test.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+			ExpectedMessage: cmdUtils.FormatAvailableResourcesMessage(),
+		},
+		{
+			Title:           "kind not managed",
+			Args:            []string{"whatever"},
+			IsErrorExpected: true,
+			ExpectedMessage: `resource "whatever" not managed`,
+		},
+		{
+			Title:           "not connected to anyAPI",
```
Comment (AntoineThebaud) [inline internal/cli/cmd/remove/remove_test.go:39]: any API

File: internal/cli/cmd/remove/remove_test.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+		{
+			Title:           "delete a single resource from a file",
+			Args:            []string{"-f", "../../utils/test/sample_resources/single_resource.json", "--project", "perses"},
+			APIClient:       fake_api.New(),
+			IsErrorExpected: false,
+			ExpectedMessage: `object "Folder" "ff15" has been deleted in the project "perses"
+`,
+		},
+		{
+			Title:           "delete multiples different resources from a file",
```
Comment (AntoineThebaud) [inline internal/cli/cmd/remove/remove_test.go:78]: multiple resources

**Review by AntoineThebaud — APPROVED** [review]
LGTM, just spotted few typos in some tests titles

**Comment by Nexucis** [comment]
cool, thanks @AntoineThebaud !

**Pull request merged**